app/src/main/res/values/strings.xml

```diff
@@ -961,6 +961,8 @@ Do you want to switch to this account?</string>
   <string name="prevent_account_lockout">Prevent account lockout</string>
   <string name="generate_master_password">Generate master password</string>
   <string name="generate_button_label">Generate</string>
-    <string name="write_this_password_down_and_keep_it_somewhere_safe">Write this password down and keep it somewhere safe.</string>
+  <string name="write_this_password_down_and_keep_it_somewhere_safe">Write this password down and keep it somewhere safe.</string>
   <string name="learn_about_other_ways_to_prevent_account_lockout">Learn about other ways to prevent account lockout</string>
+  <string name="master_password_require_message">Master password field is required</string>
+  <string name="pin_require_message">PIN field is required</string>
```

Comment:
We don't need new strings. We can use a combination of `validation_field_required` and `master_password` or `pin` resources that are already defined.

Reply:
done